### DISCUSSION TITLE
option to 'dir_up' stop modifying the cwd when called at root path

### DISCUSSION THREAD

**Discussion by otavioschwanck** [nvim-tree/nvim-tree.lua#1558] [Q&A]
Sometimes i just want to paste the file on a dir up, and it keep messing the cwd.

**Comment by alex-courtis** [comment]
There are several options that may help you achieve your desired behaviour:

`:help nvim-tree.actions.change_dir`

`:help nvim-tree.sync_root_with_cwd`

`:help nvim-tree.respect_buf_cwd`

Let me know how you go or have any further questions.
